lib/help.ts

```diff
@@ -40,6 +40,27 @@ export default class BalenaHelp extends Help {
 			return;
 		}
 
+		// If they've typed a topic (e.g. `balena os`) that isn't also a command (e.g. `balena device`)
+		// then list the associated commands.
+		const topicCommands = this.config.commands.filter((c) => {
+			return c.id.startsWith(`${subject}:`);
+		});
+		if (topicCommands.length > 0) {
+			console.log(`${chalk.yellow(subject)} commands include:`);
+			console.log(this.formatCommands(topicCommands));
+			console.log(
+				`\nRun ${chalk.cyan.bold(
+					'balena help -v',
+				)} for a list of all available commands,`,
+			);
+			console.log(
```

Comment:
Should all these `console.log` messages be `console.error` instead? I.e. using `stderr` instead of `stdout`. One "school of thought" is that any messages (including info, warn, error) are not part of the intended output of the command should go to stderr. For example, if the user typed:

```
$ balena envss > env_vars.txt
```
The intention would be for the list of env vars (printed to `stdout`) to be saved to a text file, while any info / warn / error messages would not be redirected. Using `console.error` above would mean that the messages would not be redirected to the text file. If the user wanted both stderr and stdout to go to the text file, they could use:
```
$ balena envss &> env_vars.txt
```
(At least with `bash`)